app/bridge/client.js

```diff
@@ -90,85 +121,142 @@ function createNewClient(name, version) {
     this.client = client;
     this.uuid = uuid();
 
-    this.server = null;
+    this.serverChannel = null;
```

Comment:
Yep, sorry this PR is a mess. I took too long and I should have made several of them. 

That being said, there's too many things called `server` in this app imho (there's at least 3 server.js files also). We found that difficult to deal with, especially when talking together about the app.

 Also, it makes clear to the developer that this is a channel, not a ref to an object instantiated with `new Server()` (that wasn't clear for me at the very beginning). I thought this name described more accurately what this ref is about, but I have no strong opinion (apart that I would really like it _not_ to be called `server`).